index.html

```diff
@@ -8,7 +8,9 @@
     <link rel="stylesheet" href="styles.css" />
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>E-store</title>
+    <title>E-store Demo</title>
+    <script type="module" type="text/javascript" src="index.js"></script>
+    <script type="module" type="text/javascript" src="webex.js"></script>
```

Comment:
Isn't it standard to put these scripts after the body tag so that it can read all elements just in case we have some DOM queries before `window.onload`?